service/model.py

```diff
@@ -158,6 +158,12 @@ def find_by_id_status(cls, by_id: int, by_status=True):
         cls.logger.info("Processing lookup for id %s with status %s", by_id, by_status)
         return cls.query.filter(cls.id==by_id, cls.status==by_status)
 
+    @classmethod
+    def find_by_id_and_typeid(cls, by_id: int, by_typeid: int):
```

Comment:
Great work George, can you add unit tests for this function as well? and also all of the previous functions that you have written in model.py?

Reply:
Sure! I will working on it!